test/info-extras-test.js

```diff
@@ -175,6 +176,13 @@ describe('extras.getRelatedVideos()', () => {
     assertRelatedVideos(extras.getRelatedVideos(info));
   });
 
+  describe('With richThumbnails', () => {
+    it('Returns related videos', () => {
+      const info = require('./files/videos/rich-thumbnails/expected_info.json');
+      assertRelatedVideos(extras.getRelatedVideos(info));
```

Comment:
pass an extra param here to check that `richThumbnails` is not empty

Reply:
i meant here to call it like `assertRelatedVideos(extras.getRelatedVideos(info), true)` and inside the function make sure the `richThumbnails` array for each video is not empty, since it shouldn't be empty for this test.

Reply:
Oh, ok

Reply:
done.